api/core/routes.py

```diff
@@ -174,7 +163,143 @@ def get_ephemeris_by_name_jdstep():
     if [x for x in (name, latitude, longitude, elevation, startjd, stopjd, stepjd) if x is None]:
         abort(400) 
 
-    tleLine1, tleLine2, date_collected = getTLE(name)
+    #Cast the latitude, longitude, and jd to floats (request parses as a string)
+    lat = float(latitude)
+    lon = float(longitude)
+    ele = float(elevation)
+    
+    jd0 = float(startjd)
```

Comment:
Here and throughout, I suggest we explicitly adopt astropy time objects. Being accurate and precise about our times will be extremely important down the line; here's one nice writeup of why it's better than datetime fwiw https://sunpy.org/posts/2018/2018-10-20-time

Reply:
Good point, also will be necessary if we integrate with astroplan or anything else like that. I'll work on that in a new branch and update this pull request when I merge it into develop.

Reply:
Happy to adopt astropy for Time and coordinate transforms. I am also a stark proponent of leaving a number alone if there aren't any transforms happening.